e2e/.gitignore

```diff
@@ -0,0 +1,5 @@
+/vendors
```

Comment:
I believe that was something left over when i rebased/merged, i have removed vendors from .gitignore